common/src/main/resources/Operator.properties

```diff
@@ -261,6 +261,15 @@ WLSDO-0056=Persistent volume claim {0} is invalid, the required field ''spec.res
 WLSDO-0057=Persistent volume claim {0} is invalid, the required field ''spec.storageClass'' must be specified.
 WLSDO-0058=The OPSS secrets cannot be specified in both ''{0}'' and ''{1}'' for a JRF domain.
 WLSDO-0059=The configuration ''{0}'' is not allowed when ''{1}'' is specified.
+WLSDO-0060=The mount path ''{0}'' in entry ''{1}'' and the mount path ''{2}'' in entry ''{3}'' are overlapping.
+WLSDO-0061=One of the volume mounts in the domain spec needs to contain the path for domain home ''{0}'' \
+  when ''spec.configuration.initializeDomainOnPV'' is specified.
+WLSDO-0062=The ''spec.configuration.initializeDomainOnPV.domain.createIfNotExists'' cannot be ''DOMAIN_AND_RCU'' when the \
```

Comment:
it should be domainAndRCU instead of 'DOMAIN_AND_RCU'

Reply:
fixed. thanks